---a/net/minecraft/world/border/IBorderListener.java
+++b/net/minecraft/world/border/IBorderListener.java
@@ -1,7 +1,7 @@
 package net.minecraft.world.border;
 
-public interface IBorderListener
-{
+public interface IBorderListener {
+
     void onSizeChanged(WorldBorder border, double newSize);
 
     void onTransitionStarted(WorldBorder border, double oldSize, double newSize, long time);
@@ -15,4 +15,4 @@
     void onDamageAmountChanged(WorldBorder border, double newAmount);
 
     void onDamageBufferChanged(WorldBorder border, double newSize);
-}
+}
